Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -260,7 +265,8 @@
                                         "Supplier($select=Name)," +
                                         "AssociatedSystemUsages($expand=ItSystemUsage($select=Id;$expand=ItSystem($select=Name,Disabled)))," +
                                         "DataProcessingRegistrations($select=IsAgreementConcluded)," +
-                                        "LastChangedByUser($select=Name,LastName)";
+                                        "LastChangedByUser($select=Name,LastName)," +
+                                        "CriticalityType($select=Name)";
```

Comment:
Name is not a key, only select the key.. look in other overviews how to manage this.,.. basically you want the full options set delivered from the options service.. create a lookup of that and use the map to generate "filter options", as well as resolving the "names" based on the uid

See in It-system-overview.ts or the DPR overview. They use choice types in the grid